pages/super-admin/assessment/index.tsx

```diff
@@ -81,11 +87,15 @@ function Index() {
         console.log(data);
       } catch (error) {
         console.error('Error fetching data:', error);
+      } finally {
+        setGettingData(false);
       }
     };
-
-    fetchData();
-    assessmentOverviewData();
+    const getData = async () => {
+      fetchData();
```

Comment:
What's the essence of invoking the functions in a wrapping async func if you are not going to use await?